mgmt/rest/server.go

```diff
@@ -92,6 +93,15 @@ func New(cfg *Config) (*Server, error) {
 		negroni.HandlerFunc(s.authMiddleware),
 	)
 	s.r = httprouter.New()
+
+	// Deals with CORS
+	c := cors.New(cors.Options{
+		AllowedOrigins: []string{"*"},
```

Comment:
@andrzej-k, agreed your concerns. Should we allow localhost for now until we know whoch domains are allowed?